Video.java

```diff
@@ -0,0 +1,73 @@
+import java.util.Date;
+
+public class Video {
+	private String title ;
+
+	private int priceCode ;
+	public static final int REGULAR = 1 ;
+	public static final int NEW_RELEASE =2 ;
+	
+	private int videoType ;
+	public static final int VHS = 1 ;
+	public static final int CD = 2 ;
+	public static final int DVD = 3 ;
+	
+	private Date registeredDate ;
+	private boolean rented ;
+	
+	public Video(String title, int videoType, int priceCode, Date registeredDate) {
+		this.setTitle(title) ;
+		this.setVideoType(videoType) ;
+		this.setPriceCode(priceCode) ;
+		this.registeredDate = registeredDate ;
+	}
+
+	public int getLateReturnPointPenalty() {
+		int pentalty = 0 ;
+		switch ( videoType ) {
+			case VHS: pentalty = 1 ; break ;
+			case CD: pentalty = 2 ; break ;
+			case DVD: pentalty = 3 ; break ;
+		}
+		return pentalty ;
+	}
+	public int getPriceCode() {
+		return priceCode;
+	}
+
+	public void setPriceCode(int priceCode) {
+		this.priceCode = priceCode;
+	}
+
+	public String getTitle() {
+		return title;
+	}
+
+	public void setTitle(String title) {
+		this.title = title;
+	}
+
+	public boolean isRented() {

```

Comment:
Please refactoring Rantal information in Video.java file to Rental.java file.
I think move to Rental.java is natural.